qfieldsync/ui/cloud_transfer_dialog.ui

```diff
@@ -95,8 +95,14 @@
        </property>
        <item>
         <widget class="QLabel" name="feedbackLabel_2">
+         <property name="sizePolicy">
+          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
+           <horstretch>0</horstretch>
+           <verstretch>0</verstretch>
+          </sizepolicy>
+         </property>
          <property name="text">
-          <string>The project check provides automatic feedback for some common mistakes when using QField, QFieldSync and QFieldCloud. Lines starting with ERROR show problems in your project configuration that must be fixed before proceeding. Lines starting with WARNING are suggestions what you can improve.</string>
+          <string>The project check provides automatic feedback for some common mistakes when using QField, QFieldSync and QFieldCloud. Errors show problems in your project configuration that must be fixed before proceeding. Warnings are suggestions what you can improve.</string>
```

Comment:
Should we use something similar to a :bulb: instead of the warning sign? Would look much friendlier.

Reply:
@m-kuhn , big +1 , I like it. I'll implement